documentation/_puppetdb_nav.html

```diff
@@ -60,6 +60,7 @@ <h2 id="puppetdb">PuppetDB {{ puppetdb_version }}</h2>
       <li><a href="/puppetdb/{{ puppetdb_version }}/api/query/v4/query.html">Query Structure</a></li>
       <li><a href="/puppetdb/{{ puppetdb_version }}/api/query/v4/operators.html">Available Operators</a></li>
       <li><a href="/puppetdb/{{ puppetdb_version }}/api/query/v4/paging.html">Query Paging</a></li>
+      <li><a href="/puppetdb/{{ puppetdb_version }}/api/query/v4/root.html">Root Endpoint (Experimental)</a></li>
```

Comment:
Oh nice sorry, misunderstood. Is there a way we could redirect `/puppetdb/{{ puppetdb_version }}/api/query/v4` to `/puppetdb/{{ puppetdb_version }}/api/query/v4/root.html`? Otherwise the name is :+1:  from me.